examples/tina-cloud-starter/pages/posts/[filename].tsx

```diff
@@ -34,11 +34,11 @@ export const query = `#graphql
     getPostsDocument(relativePath: $relativePath) {
       data {
         __typename
-        ... on Article_Doc_Data {
+        ... on Article_Data {
```

Comment:
Here's the crux of the breaking change, unfortunately GraphQL doesn't let you create aliases. We could get around this by duplicating the types, and perhaps that's the easiest path forward for the short term.